content/06.methods.md

```diff
@@ -254,6 +254,15 @@ If the copy number value was 0, we set the status to "deep deletion".
 For autosomes only, we set the status to "amplification" when the copy number value was greater than two times the ploidy value.
 We plotted genome-wide gains and losses in (**Figure {@fig:S3}B**) using the R package `ComplexHeatmap` [@doi:10.1093/bioinformatics/btw313].
 
+#### Breakpoint Density (`chromosomal-instability` analysis module)
+
+We defined breakpoint density as the number of breaks per genome or exome per sample.
+For Manta SV calls, we filtered to retain "PASS" variants and used breakpoints from the algorithm.
+For consensus CNV calls, if |log<sub>2</sub> ratio| > log<sub>2</sub>(1 + 0.5), we annotated the segment as a break. 
```

Comment:
Can you point me to where you got `0.5` here please?

Reply:
Ah, I think I may have been looking at the wrong thing - the gap, but I see this default option for percent change is 0, so I will update this.